src/components/tabs/index.tsx

```diff
@@ -47,16 +48,29 @@ const Tabs = ({
 	const tabItems = useTabItems({ children, activeTabIndex, initialActiveIndex })
 
 	/**
-	 * useSyncedTabGroups hooks into TabProvider,
-	 * and keeps activeTabIndex & activeTabGroup in sync.
+	 * Handle syncing `activeTabIndex` and `activeTabGroup`.
 	 *
 	 * Note: only works where tabs have groups, eg <Tab group="some-string" />.
 	 */
-	const setSyncedActiveTabIndex = useSyncedTabGroups({
```

Comment:
❓ question: I'm curious why you opted to remove `useSyncedTabGroups`? It felt like a nice encapsulation of this logic

Reply:
That code wasn't worked as expected, so when I tried out something else in the body of the `Tabs` component I left code there and decided not to move it into `useSyncedTabGroups`. Updated to move the new code into that file in https://github.com/hashicorp/dev-portal/pull/1077/commits/a14ed10d49c74ce9be546d426fb696dda113529a.